caf_solution/local.database.tf

```diff
@@ -24,6 +24,7 @@ locals {
       mssql_servers                      = var.mssql_servers
       mysql_databases                    = var.mysql_databases
       mysql_servers                      = var.mysql_servers
+      postgresql_servers                 = var.postgresql_flexible_servers
```

Comment:
maybe better: postgresql_flexible_servers = var.postgresql_flexible_servers

Reply:
@arnaudlh  Fixed the typo 1629b2aa